src/utils/render-fragments.js

```diff
@@ -1,13 +1,152 @@
 import React from 'react'
 import scUrls from './../../_data/sc-urls'
-import pkg from './../../package.json'
 import { Link } from 'gatsby'
+import glossaryUsages from './../../_data/glossary-usages.json'
 
-export function getAccessibilityRequirements(accessibility_requirements) {
+export const getChangelog = (changelog, url, file) => {
+	if (!changelog.length) {
+		return null
+	}
+	return (
+		<>
+			<br />
+			<hr />
+			<a id="changelog" href="#changelog">
+				<h2>Changelog</h2>
+			</a>
+			<ul>
+				{changelog.map(log => {
+					const { commit: hash, msg, date } = log
+					const versionUrl = `${url}/blob/${hash}/${file}`
+					const changesUrl = `${url}/commit/${hash}`
+
+					return (
+						<li key={hash}>
+							{getDateTimeFromUnixTimestamp(date)} - {msg}
+							&nbsp;
+							<a target="_blank"
+								rel="noopener noreferrer"
+								href={versionUrl}>
+								See version
+							</a>
+							&nbsp;
+							<a target="_blank"
+								rel="noopener noreferrer"
+								href={changesUrl}>
+								See changes
+							</a>
```

Comment:
Now that I'm seeing these links... not so pretty :/. Can we align these and use icons?

Reply:
A table is better placed for alignment, and I swapped it to icons.

![image](https://user-images.githubusercontent.com/20978252/59097727-3734be00-8916-11e9-90c7-5b68bef1ac49.png)